gamestate.py

```diff
@@ -389,6 +451,16 @@ def test_types(player, pokemon, type_names):
 			gs.set_type(player, pokemon, type_name)
 		check_types(player, pokemon, type_names)
 	
+	def check_statuses(player, pokemon, status_names):
```

Comment:
These test functions never actually seem to be used.

Reply:
Correct! Working on it

Reply:
Resolved